web/docs/setup-vast/deploy/aws.md

```diff
@@ -225,3 +225,18 @@ datasource resources only, use:
   bucket when the stack is destroyed to avoid interfering with your existing
   notification systems.
 :::
+
+### Run MISP
+
+[MISP](https://www.misp-project.org/) is a popular community driven open source
+thread intelligence platform. It integrates naturally with VAST. To deployed a
+preconfigured instance:
+- add `misp` to the list of plugins in `VAST_CLOUD_PLUGINS`
+- deploy (or re-deploy) the VAST stack including MISP with `./vast-cloud deploy`
+- start the MISP server with `./vast-cloud misp.start`, wait a few minutes, MISP
```

Comment:
Dot notation in CLI was a bit unexpected. Would it be better to have something like `--misp-start`, `--misp-stop` ? Also in the future adding new commands would cause overload of commands that could be confusing. Maybe we can align on a template for commands for services and have only a template for them in documentation. Ex:

```
For services in [misp, ex1, ex2] following commands can be used:
{service}.start
{service}.stop
{service}.status
...
```

I am aware that CLI libraries do not allow many customizations but at least would like to discuss about a template.

Reply:
I am also not fond of this dot notation. To me whitespace would be the most logical. But I couldn't find an easy way to achieve this with the PyInvoke library yet.

> Also in the future adding new commands would cause overload of commands that could be confusing

not sure what you mean

> Maybe we can align on a template for commands for services and have only a template for them in documentation.

Interesting point. Right now the VAST server service does not comply to the `{service}.xxxx` pattern, but I think that it would be better if it did :-)

Reply:
created a [story](https://app.shortcut.com/tenzir/story/37554/make-all-cloud-service-commands-obey-the-same-pattern) to track it to avoid snowballing in this PR

Regarding the dot notation, I don't have a good alternative solution and I think it's good enough for now.